src/components/Navbar.css

```diff
@@ -0,0 +1,72 @@
+.navbar {
+  background-color: #060b26;
+  height: 80px;
+  display: flex;
+  justify-content: start;
+  align-items: center;
+}
+
+.menu-bars {
+  margin-left: 2rem;
+  font-size: 2rem;
+  background: none;
+}
+
+.nav-menu {
+  background-color: #060b26;
+  width: 250px;
```

Comment:
Same with this width stuff, imo relative widths (like vw or percent or rem or whatever) is better than hard pixel codings?? But I'm not married to this rule

Reply:
Yeah I agree for most components, just except for things that are supposed to be constant like the sidebar, toolbar, etc